tarantool-lsp/tnt-doc/doc-parser.lua

```diff
@@ -47,15 +47,11 @@ local function parseFunction(scope, moduleName)
     end
     termDescription = rtrim(ltrim(termDescription or ""))
 
-    if moduleName and not funcName:find(moduleName) then
```

Comment:
Why? It's a special heuristic for more informative output.

Reply:
It doesn't add any sense right now.

Reply:
???
Why you just remove it?

Reply:
Because it produce duplicates.